modular_skyrat/master_files/code/modules/jobs/off_duty_check.dm

```diff
@@ -0,0 +1,51 @@
+/**
+ * Checks is the parent mob is currently off duty. Returns `FALSE` if the mob isn't clocked out.
+ *
+ * Arguements
+ * * `whitelisted_deparments` - This list contains the paths of `/datum/job_department`s we want to check if the parent mob is part of.
+ * If present, we will only check if the mob is off-duty if they are a member of one of the departments in the list
+ * Otherwise, if we have nothing in the list, we will check all departments
+ * * `blacklisted_jobs` - This is a list that contains paths of the `/datum/job`s that we will always return `TRUE` for
+ * This typically applies for jobs like assistant where clock status probably wouldn't really matter.
+*/
+
+/mob/living/proc/check_if_off_duty(list/whitelisted_deparments = list(), list/blacklisted_jobs = list(/datum/job/assistant))
```

Comment:
Any reason why this has to be on the `/living` level? Like, wouldn't we be better off having it on the `/human` level? I can't really see a usecase where we want to check if any random mob is actually off-duty.